server/tinylicious/package-lock.json

```diff
@@ -190,7 +190,7 @@
       "integrity": "sha512-H+wEaxuIHODVNqyY8XSMY6ww7ndrRfht9CXKUAUzdQjUN1Oi++YonKcD3CXWZod6afxZ6abDmltIO9wLrjOJzg=="
     },
     "@fluidframework/common-utils": {
-      "version": "0.26.0",
+      "version": "0.27.0",
```

Comment:
don't run find replace on the lock files, do just the package.jsons then run the install and these will be updated automatically